envire_maps.orogen

```diff
@@ -93,3 +93,39 @@ typekit do
                  *envire_maps
 end
 
+# Abstract map laoder for MLS maps.
+task_context "MLSMapLoader" do
+
+    property('path', 'std/string')
+    .doc("File location of the PLY to load.")
+
+    property('resolution', 'double', 0.1)
+    .doc("Cell size of the generated MLS map.")
+
+    property('mls_config', 'maps/grid/MLSConfig')
+    .doc("Parameters of the generated MLS map.")
+
+    property('map_frame', 'std/string', "static_map")
+    .doc("Envire frame the map will be attached to.")
+
+    operation('publishMap')
```

Comment:
I guess it would be nice if this method would return a boolean.

Reply:
Why? It just sends the map. If loading fails, the map will fail during configure already.

Reply:
Because it evaluates the member `mMapLoaded`. If you can guarantee that this is always true it is not necessary to evaluate it.